horovod/tensorflow/__init__.py

```diff
@@ -520,7 +535,9 @@ def allreduce_grads(grads, vars=None):
                                                                op=op,
                                                                prescale_factor=prescale_factor,
                                                                postscale_factor=postscale_factor,
-                                                               process_set=process_set)
+                                                               process_set=process_set,
+                                                               name=f"grad_{i}" if use_generic_names else None,
+                                                               ignore_name_scope=True if use_generic_names else False)
```

Comment:
is there any reason we don't do `ignore_name_scope=use_generic_names` given that both are booleans?

Reply:
No particular reason. Your suggestion is cleaner so I made that change. Thanks!